libarchive/archive_read_support_format_tar.c

```diff
@@ -1036,16 +1036,14 @@ header_Solaris_ACL(struct archive_read *a, struct tar *tar,
 	int64_t type;
 	char *acl, *p;
 
-	/*
-	 * read_body_to_string adds a NUL terminator, but we need a little
-	 * more to make sure that we don't overrun acl_text later.
```

Comment:
This is apparently an old comment.  At least, it doesn't make sense now, so I deleted it.